Commands/CommandAll.cs

```diff
@@ -27,12 +27,12 @@ void ICommand.Execute(Context context, string command, string[] parameters)
             context.GetCurrent().GetName();
             if (GuardEq(parameters, 1))
             {
-                Console.WriteLine("I don't seem to know where that is 🤔");
+                Console.WriteLine("Jeg kan ikke finde ud af hvor det er henne? 🤔");
                 return;
             }
             switch (parameters[0])
             {
-                case "Tilføje":
+                case "Tilføj":
```

Comment:
Det er fint, men huske at du også skal rette i de valg, som brugeren for vist længere oppe